app/src/main/res/values/strings.xml

```diff
@@ -178,15 +178,15 @@
     <string name="ivtext_title">Your lvl %1$s, CP %2$s, HP %3$s %4$s can be:</string>
     <string name="ivtext_possibilities">%1$s more possibilities...</string>
     <string name="ivtext_no_possibilities">No possibilities, please check your stats again!</string>
-    <string name="ivtext_iv">Min: %1$s%% Average: %2$s%% Max: %3$s%%\n</string>
+    <string name="ivtext_iv">Min: %1$.2f%% Average: %2$.2f%% Max: %3$.2f%%\n</string>
     <string name="ivtext_evolve">Can be evolved into %1$s:\n</string>
     <string name="ivtext_evolve_further">Can be further evolved into %1$s:</string>
     <string name="ivtext_many_possibilities">There are too many possibilities for this pokemon. Try powering it up!</string>
     <string name="ivtext_max_lvl_cost">Cost to reach lvl %1$s:</string>
     <string name="ivtext_max_lvl_cost2">Candy: %1$s Stardust: %2$s</string>
     <string name="main_battery_saver">Battery Saver (Screenshot to Scan)</string>
     <string name="ivtext_cp_lvl">CP at lvl %1$s: %2$s - %3$s</string>
-    <string name="ivtext_stats">Atk: %1$02d Def: %2$02d Sta: %3$02d — %4$3d%%</string>
+    <string name="ivtext_stats">Atk: %1$02d Def: %2$02d Sta: %3$02d — %4$.2f%%</string>
```

Comment:
Also, however many decimals we use here, it's also good to keep specifying a width to ensure alignment.